documentation/book/oauth/proc-oauth-kafka-config.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following module:
+//
+// con-oauth-config.adoc
+
+[id='proc-oauth-kafka-config-{context}']
+= Configuring OAuth for Kafka components
+
+This procedure describes how to configure Kafka components to use {oauth} authentication using an authorization server.
```

Comment:
For further discussion: we should agree on terminology when describing Kafka clients and Kafka components in the doc. Just something to consider.

Reply:
Yes.